app/models/form_handler.rb

```diff
@@ -198,6 +207,22 @@ def sales_earliest_open_for_editing_collection_start_date(now: Time.zone.now)
     end
   end
 
+  def years_of_available_lettings_forms
+    years = []
+    lettings_forms.each_value do |form|
```

Comment:
can we combine these two functions into one that takes the forms as input?

Reply:
Do you mean something like this?
`FormHandler.instance.years_of_available_forms(FormHandler.instance.lettings_forms)` 

I feel like it might be more confusing than splitting these methods up, let me know if you meant something different